scipy/signal/tests/test_spectral.py

```diff
@@ -84,7 +84,7 @@ def test_integer_twosided(self):
     def test_complex(self):
         x = np.zeros(16, np.complex128)
         x[0] = 1.0 + 2.0j
-        f, p = periodogram(x)
+        f, p = periodogram(x, return_onesided=False)
```

Comment:
Why did you change all of these? You shouldn't need to, right? The default doesn't appear to have changed.